backend/src/apis/IotaConnector.ts

```diff
@@ -0,0 +1,103 @@
+import { Client, ClientBuilder } from '@iota/client'
+import { MessageWrapper } from '@iota/client/lib/types'
+
+import { CONFIG } from '@/config'
+import { backendLogger as logger } from '@/server/logger'
+
+// Source: https://refactoring.guru/design-patterns/singleton/typescript/example
+// and ../federation/client/FederationClientFactory.ts
+/**
+ * A Singleton class defines the `getInstance` method that lets clients access
+ * the unique singleton instance.
+ */
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
+class IotaClientSingleton {
+  private static IotaClint: Client
```

Comment:
Bitte schreibe Client korrekt
```suggestion
  private static IotaClient: Client
```